src/Network/WebSockets/Connection.hs

```diff
@@ -252,6 +255,7 @@ data Connection = Connection
     { connectionOptions   :: !ConnectionOptions
     , connectionType      :: !ConnectionType
     , connectionProtocol  :: !Protocol
+    , connectionHeartbeat :: !(MVar ())
```

Comment:
I would add a comment to the field explaining how this is used (or refer to the PingPong module).

Reply:
Done :)